src/react/components/openSearchModalButton.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
```

Comment:
When and where does the modal come into play? I did not see it on the demo site.

Reply:
@sherakama The modal is being developed in a separate ticket (ADAPT-3247). 
This file probably shouldn't be in this PR at all... I must have accidentally committed code to this branch on accident. I'll get that cleaned up.